src/main/java/org/folio/search/cql/CqlTermQueryConverter.java

```diff
@@ -89,7 +91,16 @@ public QueryBuilder getQuery(CQLTermNode termNode, String resource) {
         "Failed to parse CQL query. Comparator '%s' is not supported.", comparator));
     }
 
+    var modifiers = termNode.getRelation().getModifiers().stream()
+      .map(Modifier::getType)
+      .toList();
+
     if (CollectionUtils.isNotEmpty(fieldsList)) {
+      if (modifiers.contains(STRING_MODIFIER)) {
+        var updatedFieldsList = getUpdatedFields(fieldsList);
+        return termQueryBuilder.getQuery(searchTerm, resource, updatedFieldsList.toArray(String[]::new));
+      }
+
```

Comment:
In such implementation, you modify all possible operators, but this `string` modifier could work and impact on query only for the "==" operator. 
I expect that `TermQueryBuilder.getQuery` method will be modified to have a new parameter with a list of modifiers. It will help to separate logic of query creation for each operator and will open opportunities for future implementation for other modifiers.